gamemodes/terrortown/gamemode/shared/sh_sprint.lua

```diff
@@ -192,9 +101,7 @@ end
 -- @param table modifierTbl The table in which the modifier can be changed
 -- @hook
 -- @realm shared
-function GM:TTT2StaminaRegen(ply, modifierTbl)
-
-end
+function GM:TTT2StaminaRegen(ply, modifierTbl) end
```

Comment:
Hm, while this works as well, we always defined empty functions in a different way. I guess it doesn't matter but I don't get why you changed it anyway?